packages/xo-web/src/common/utils.js

```diff
@@ -550,15 +550,21 @@ export const getIscsiPaths = pbd => {
 
 // ===================================================================
 
-export const downloadLog = ({ log, date, type }) => {
-  const file = new window.Blob([log], {
+export const createBinaryFile = str =>
+  new window.Blob([str], {
     type: 'text/plain',
   })
+
+// ===================================================================
+
+export const formatDate = ms => new Date(ms).toISOString().replace(/:/g, '_')
```

Comment:
This is only one way of formatting a file, what is specific to this one?

Reply:
I just mutualized the generated date between the downloaded log and the sent log. 

https://github.com/vatesfr/xen-orchestra/pull/4201/files#diff-822ef9925d865be023259e6060ce809cL559